sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosScripts.java

```diff
@@ -226,18 +237,18 @@ public CosmosTrigger getTrigger(String id) {
      * @return the cosmos sync stored procedure response
      * @throws CosmosClientException the cosmos client exception
      */
-    CosmosStoredProcedureResponse mapStoredProcedureResponseAndBlock(Mono<CosmosAsyncStoredProcedureResponse> storedProcedureResponseMono)
-            throws CosmosClientException {
+    CosmosStoredProcedureResponse mapStoredProcedureResponseAndBlock(
+        Mono<CosmosAsyncStoredProcedureResponse> storedProcedureResponseMono) throws CosmosClientException {
         try {
             return storedProcedureResponseMono
-                           .map(this::convertResponse)
-                           .block();
+                       .map(this::convertResponse)
+                       .block();
         } catch (Exception ex) {
             final Throwable throwable = Exceptions.unwrap(ex);
             if (throwable instanceof CosmosClientException) {
                 throw (CosmosClientException) throwable;
             } else {
-                throw ex;
+                throw (RuntimeException) ex;
```

Comment:
What if the exception is a checked exception ?
We cannot cast it directly to `RuntimeException` without checking. 
I thought that was the reason we used `throw Exceptions.propagate(ex)` at other places.

Reply:
Reverted